.github/workflows/build-deploy-dev.yml

```diff
@@ -54,7 +54,7 @@ jobs:
         working-directory: "./devops"
         run: |
           make oc-build-workers
-      - name: Build Queues Image
+      - name: Build Queue Consumers Image
```

Comment:
There are descriptions like `Build Web, SIMS-Api and Workers` and `Deploy Web, SIMS-Api and Workers` that can be adapted. I am not sure if we added the `queue-consumers` or just stop mentioning them individually.